OMF_API/AppSettings.cs

```diff
@@ -0,0 +1,116 @@
+﻿using System.Collections.Generic;
+
+namespace OMF_API
+{
+    public class AppSettings
+    {
+        /// <summary>
+        /// The list of endpoints to connect and send to
+        /// </summary>
+        public IList<Endpoint> endpoints { get; set; }
+    }
+
+    public class Endpoint
+    {
+        /// <summary>
+        /// The endpoint type. This will be OCS, EDS, or PI
+        /// </summary>
+        public string endpoint_type { get; set; }
```

Comment:
Any particular reason we used snake_case for these? I think we typically use PascalCase in our settings files.

Reply:
I went for snake case since that's the way I did it for Python and I wanted to use the same appsettings file in both. I can change this, though

Reply:
Ahh. Snake case is good for variables and methods in Python, but I think I'd just use PascalCase in the JSON files. Thanks!